JitsiConference.js

```diff
@@ -1348,21 +1350,21 @@ JitsiConference.prototype._setupNewTrack = function(newTrack) {
 
 /**
  * Sets the video type.
- * @param track
+ * @param trackVideoType
  * @return <tt>true</tt> if video type was changed in presence.
  * @private
  */
-JitsiConference.prototype._setNewVideoType = function(track) {
+JitsiConference.prototype._setNewVideoType = function(trackVideoType) {
```

Comment:
can you keep the track argument here? with multi stream support the track will be needed here

Reply:
done, but the case with going from screen share back to video muted is handled only for presence, the case when there is no video track.